alpheios_nemo_ui/data/assets/scss/cookies.scss

```diff
@@ -30,4 +30,16 @@
             text-decoration: none;
         }
     }
+}
+
+#browser-supported-banner {
+    bottom: 50%;

```

Comment:
can we move it closer to the top of the page? I think it looks a little strange in the middle

Reply:
I have placed it 20% from the top on mobile and 10% on tablet/desktop

![image](https://user-images.githubusercontent.com/12377640/62465697-dec55400-b7d2-11e9-9006-eac1690eec7e.png)

![image](https://user-images.githubusercontent.com/12377640/62465826-2d72ee00-b7d3-11e9-9d8e-9d518da9ad96.png)